state/payloads.go

```diff
@@ -100,6 +101,7 @@ func (UnitPayloads) LookUp(name, rawID string) (string, error) {
 // is already in the DB then it is replaced.
 func (up UnitPayloads) Track(pl payload.Payload) error {
 
+	// anastasiamac: not sure why is this a problem if the unit is known here?...
```

Comment:
I did :(... but I also felt it was a separate PR... there were also other eyebrow-raising places and comments that I wish were not in production code... 
I have originally removed them in my big payload-move PR (now closed and abandoned)... they will all need to be addressed separately from here.